### DISCUSSION TITLE
Lib File error for part 6

### DISCUSSION THREAD

**Discussion by nuprince** [Cyfrin/foundry-full-course-cu#156] [General]
when using the forge init command ... all but the lib file downloads into the directory .. and i'm getting this error in the terminal 
Any suggestions .... 

'''warning: could not open directory 'Library/Application Support/MobileSync/': Operation not permitted
warning: could not open directory 'Library/Application Support/CallHistoryTransactions/': Operation not permitted
warning: could not open directory 'Library/Application Support/com.apple.sharedfilelist/': Operation not permitted
warning: could not open directory 'Library/Application Support/com.apple.TCC/': Operation not permitted
warning: could not open directory 'Library/Application Support/FileProvider/': Operation not permitted
warning: could not open directory 'Library/Application Support/AddressBook/': Operation not permitted
warning: could not open directory 'Library/Application Support/CallHistoryDB/': Operation not permitted
warning: could not open directory 'Library/Autosave Information/': Operation not permitted
warning: could not open directory 'Library/Saved Application State/com.installbuilder.appinstaller.uninstall.savedState/': Permission denied
warning: could not open directory 'Library/Saved Application State/com.installbuilder.appinstaller.savedState/': Permission denied
warning: could not open directory 'Library/IdentityServices/': Operation not permitted
warning: could not open directory 'Library/Calendars/': Operation not permitted
warning: could not open directory 'Library/Messages/': Operation not permitted
warning: could not open directory 'Library/HomeKit/': Operation not permitted
warning: could not open directory 'Library/Sharing/': Operation not permitted
warning: could not open directory 'Library/Mail/': Operation not permitted
warning: could not open directory 'Library/Safari/': Operation not permitted
warning: could not open directory 'Library/Suggestions/': Operation not permitted
warning: could not open directory 'Library/Containers/com.apple.VoiceMemos/': Operation not permitted
warning: could not open directory 'Library/Containers/com.apple.archiveutility/': Operation not permitted
warning: could not open directory 'Library/Containers/com.apple.Home/': Operation not permitted
warning: could not open directory 'Library/Containers/com.apple.Safari/': Operation not permitted
warning: could not open directory 'Library/Containers/com.apple.iChat/': Operation not permitted
warning: could not open directory 'Library/Containers/com.apple.CloudDocs.MobileDocumentsFileProvider/': Operation not permitted
warning: could not open directory 'Library/Containers/com.apple.mail/': Operation not permitted
warning: could not open directory 'Library/Containers/com.apple.news/': Operation not permitted
warning: could not open directory 'Library/Containers/com.apple.stocks/': Operation not permitted
warning: could not open directory 'Library/PersonalizationPortrait/': Operation not permitted
warning: could not open directory 'Library/Metadata/CoreSpotlight/': Operation not permitted
warning: could not open directory 'Library/Metadata/com.apple.IntelligentSuggestions/': Operation not permitted
warning: could not open directory 'Library/Cookies/': Operation not permitted
warning: could not open directory 'Library/Caches/com.apple.safaridavclient/': Operation not permitted
warning: could not open directory 'Library/Caches/com.apple.HomeKit.configurations/': Operation not permitted
warning: could not open directory 'Library/Caches/CloudKit/': Operation not permitted
warning: could not open directory 'Library/Caches/com.apple.Safari/': Operation not permitted
warning: could not open directory 'Library/Caches/com.apple.Safari.SafeBrowsing/': Operation not permitted
warning: could not open directory 'Library/Caches/com.apple.ap.adprivacyd/': Operation not permitted
warning: could not open directory '.Trash/': Operation not permitted
warning: CRLF will be replaced by LF in .atom/packages/etheratom/node_modules/number-to-bn/dist/number-to-bn.js.
The file will have its original line endings in your working directory
error: '.cargo/registry/index/github.com-1ecc6299db9ec823/' does not have a commit checked out
fatal: adding files failed ''

**Comment by ChefAharoni** [comment]
Where is the project directory? 

There's a chance that you're working on a directory you don't have permissions to edit.

**Reply by nuprince** [reply]
ummm foundry-simple-storage-f23 is the directory .... its also giving me a git add
 error 128 ...

**Reply by ChefAharoni** [reply]
What's the full directory?

**Comment by cromewar** [comment]
Be sure you install foundry on a user profile you have root (admin) access. Also be sure your project is on folder which has also access to Library on your Mac.
